scripts/rollup/validate/eslintrc.cjs.js

```diff
@@ -53,7 +53,7 @@ module.exports = {
     IS_REACT_ACT_ENVIRONMENT: 'readonly',
   },
   parserOptions: {
-    ecmaVersion: 5,
+    ecmaVersion: 2020,
```

Comment:
This is unfortunate that we lint it with a newer version since ideally this would catch if some uncompiled feature is included but we have to rely on closure to cover that. There's no way to configure that we just want to allow `import(...)` but not other language features.